### PR TITLE
Create landing page

### PR DESCRIPTION
**Pull request by heathermoses**
Ignore the branch name, I created the landing page rather than the trip registration page.

### CODE REVIEW

File: client/src/assets/stylesheets/App.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.hero {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 0.25)), url("../images/homepage-hero.png");
```
Comment (cpt2162) [inline client/src/assets/stylesheets/App.css]: could this be done with Tailwind? similar to how the login background image is

File: client/src/index.js [hunk 2]
```diff
@@ -2,12 +2,19 @@ import React from 'react';
...
 import Login from './pages/Login';
+import Home from './pages/Home'
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <Login />
+    <BrowserRouter>
```
Comment (cpt2162) [inline client/src/index.js:12]: nice we have multiple pages lol

File: client/tailwind.config.js [hunk 3]
```diff
@@ -8,6 +8,11 @@ module.exports = {
         'circle_logo': "url('./assets/images/circle_logo.png')"
       }
     },
+    colors: {
```
Comment (cpt2162) [inline client/tailwind.config.js:12]: This makes life easier

File: client/src/pages/Home.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import '../assets/stylesheets/App.css';
+import Navbar from '../components/Navbar';
+
+const Home = () => {
```
Comment (cpt2162) [inline client/src/pages/Home.js:4]: Looks good!

File: client/src/assets/stylesheets/App.css (hunk 1 above)
Comment (heathermoses) [reply client/src/assets/stylesheets/App.css]: Good call, I will fix that!

File: client/tailwind.config.js [hunk 5]
```diff
@@ -5,9 +5,15 @@ module.exports = {
     extend: {
       backgroundImage: {
         'login': "url('./assets/images/login-background.png')",
-        'circle_logo': "url('./assets/images/circle_logo.png')"
+        'circle_logo': "url('./assets/images/circle_logo.png')",
```
Comment (cpt2162) [inline client/tailwind.config.js:8]: looks good

**Review by cpt2162 — APPROVED** [review]

**Pull request merged**